README.md

```diff
@@ -33,12 +33,10 @@ To get more help on the Angular CLI use `ng help` or go check out the [Angular C
 
 ## Packaging 
 
-# for use in npm scripts
-npm install electron-packager --save-dev
+# Note regarding packaging
+Currently it is not possible to build Mac packages under other OS than Mac OS's. 
+It is recommended to build Windows packages under Windows OS, even if could be done
+under Linux with Wine. 
 
-# for use from cli
-npm install electron-packager -g
+Due to incompatibilities wirh WIndows OS, electron-installer-debian is not specified in package.json. It will be automated in next versions. Meanwhile, please install it by hand (npm install --save-dev electron-installer-debian) before creating Linux packages.
```

Comment:
Typo `WIndows`. Also the lines should wrap at a reasonable column to make it easier to edit.

Reply:
ok